roles/pip/tasks/main.yml

```diff
@@ -6,22 +6,18 @@
 
   - name: install pip (CentOS)
     yum:
-      name: python-pip
+      name: 
+        - python-pip
+        - python-typing
```

Comment:
What is `python-typing` for?

Reply:
its a requirement for the packages installed with pip later in this file